EXOSIMS/Prototypes/PlanetPhysicalModel.py

```diff
@@ -37,6 +37,17 @@ def __init__(self, cachedir=None, **specs):
         # load the vprint function (same line in all prototype module constructors)
         self.vprint = vprint(specs.get('verbose', True))
         
+        #Select which Phase Function to use

```

Comment:
There needs to be error checking on this input (whichPlanetPhaseFunction).  You need to force compliance with whatever typing your choose to require.